pygmt/datasets/earth_relief.py

```diff
@@ -40,8 +48,46 @@ def load_earth_relief(resolution="01d", registration=None):
         The Earth relief grid. Coordinates are latitude and longitude in
         degrees. Relief is in meters.
 
+    Notes
+    -----
+    The DataArray doesn's support slice operation, for Earth relief data with
+    resolutions higher than "05m", which are stored as smaller tiles.
+
+    Examples
+    --------
+
+    >>> # load the default grid (pixel-registered 01d grid)
+    >>> grid = load_earth_relief()
+    >>> # load the 30m grid with "gridline" registration
+    >>> grid = load_earth_relief("30m", registration="gridline")
+    >>> # load high-resolution grid for a specific region
+    >>> grid = load_earth_relief(
+    ...     "05m", region=[120, 160, 30, 60], registration="gridline"
+    ... )
+
     """
-    _is_valid_resolution(resolution)
+
+    # earth relief data stored as single grids for low resolutions
+    non_tiled_resolutions = [
+        "01d",
+        "30m",
+        "20m",
+        "15m",
+        "10m",
+        "06m",
+    ]
+    # earth relief data stored as tiles for high resolutions
+    tiled_resolutions = [
+        "05m",
+        "04m",
+        "03m",
+        "02m",
+        "01m",
+        "30s",
+        "15s",
+        "03s",
+        "01s",
+    ]
```

Comment:
When I run `black` on these two lists, they flatten into a single line. The "Style Checks" pass though, so I find it strange.

Reply:
It's automatically formatted by `black` locally, and I have black 19.10b0.

Reply:
```suggestion
    # earth relief data stored as single grids for low resolutions
    non_tiled_resolutions = ["01d", "30m", "20m", "15m", "10m", "06m"]
    # earth relief data stored as tiles for high resolutions
    tiled_resolutions = ["05m", "04m", "03m", "02m", "01m", "30s", "15s", "03s", "01s"]
```

Well, `non_tiled_resolution` is definitely under 88 characters, `tiled_resolution` is at exactly 88 characters :confused:

Reply:
Your changes also can pass the black test. 

What's your black version?

Reply:
My editor uses the system installed one which is `19.3b0`, but I have `19.10b0` installed too.